models/Likes.js

```diff
@@ -0,0 +1,68 @@
+const {Model, DataTypes } = require("sequelize");
+
+const sequelize = require("../config/connection.js");
+
+class Likes extends Model {}
+
+Likes.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    movie_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      references: {
+        model: "movies",
+        key: "id",
+      },
+    },
+    isLiked: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+    },
+    // title: {
+    //   type: DataTypes.STRING,
+    //   allowNull: false,
+    //   references: {
+    //     model: "movies",
+    //     key: "name",
+    //   },
+    // },
+    // description: {
+    //   type: DataTypes.STRING,
+    //   allowNull: false,
+    //   references: {
+    //     model: "movies",
+    //     key: "description",
+    //   },
+    // },
+    // img: {
+    //   type: DataTypes.STRING,
+    //   allowNull: false,
+    //   references: {
+    //     model: "movies",
+    //     key: "img",
+    //   },
+    // },
+    // user_id: {
```

Comment:
The user id is still needed here I think, we need to know who is liking the movie